docs/sources/writing-guide/topic-types/tutorial/index.md

```diff
@@ -96,6 +96,54 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
 
 If getting access to the tutorial data is complex, include the instructions in the steps of the tutorial. If getting access to the data is straightforward, include it in the "Before you begin" section.
 
+## Where to locate your tutorial
+
+The source for your tutorial should be in your project repo in a `tutorials` folder but it should be displayed on the main Grafana [Tutorials](/tutorials) page.  The source must be in your project repo to makes it easy for team members to review and edit the content. 
+
+Follow the steps below to hide the tutorial from your documentation's table of contents and display it on the Tutorials page.
+
+> Note: This procedure is for writers who have permissions to update the Grafana website repo.
+
+To publish your tutorial to the main Tutorials page:
+
+1. In your documentation repo, create a folder under `sources` called `tutorials` and add your tutorial to this folder.
+
+1. To hide your tutorial from your documentation table of contents, integrate the following to the `$.cascade` field in the YAML metadata of your `sources/_index.md` file. 
```

Comment:
structurally, this seems like a different task. I'd consider putting this in its own heading.